docs/4-entry-points/5-voting.md

```diff
@@ -14,16 +14,16 @@ In particular, the first version of the protocol uses a commit-reveal mechanism.
 - **State transitions:**
     - Save the controller address
 
-### 4.5.2. Set representative
+### 4.5.2. Delegate
 
-- **Actor:** Any guardian that could potentially be drafted for an adjudication round
+- **Actor:** Any guardian that could potentially be drafted for an adjudication round or an external entity incentivized in a guardian of the Protocol
```

Comment:
Similar to #26, I don't understand what "an external entity incentivized in a guardian of the Protocol" means.

Technically, any account is allowed to set a delegate here, so we could probably leave this as just "open".